src/System.Private.ServiceModel/tests/Scenarios/Bridge/.nuget/NuGet.Config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <solution>
+    <add key="disableSourceControlIntegration" value="true" />
+  </solution>
+</configuration>
```

Comment:
We should not check-in any binaries. Let's find a way to restore NuGet.exe and packages.  Note: we definitely need to try building this in the TFS mirror.  There is an explicit pass to restore build tools, and we need to ensure any other restoration logic triggers prior to doing the actual build.